docs/conf.py

```diff
@@ -59,45 +58,68 @@
 
 # To perform a Sphinx version check that needs to be more specific than
 # major.minor, call `check_sphinx_version("X.Y.Z")` here.
-check_sphinx_version("1.2.1")
+check_sphinx_version("1.2.1")  # noqa: F405
```

Comment:
Sorry for the extra PEP 8 stuff here. I needed to clean warnings off my Emacs flake8 linter so I can be sure I didn't accidentally introduce undefined vars, etc.

Reply:
Can we add explicit import for `intersphinx_mapping` instead ? Or just `intersphinx_mapping = None` before the import. I agree this is annoying, but we need the `from *` import to get all the other variables.

Reply:
Are you saying I should back out all the `# noqa`?

Reply:
If the variables are imported explicitly or defined, then you would not need the noqa everywhere.

Reply:
Yes... perhaps I'll do that in the next round of edit. 😸